### DISCUSSION TITLE
Does this project support PWA offline

### DISCUSSION THREAD

**Discussion by dohomi** [nextauthjs/next-auth#3701] [Help]
### Question 💬

I currently set up PWA and wonder if next-auth works if the app looses the connection for a short period of time. I could not find any resources of this.

### How to reproduce ☕️

`next-pwa` installed for offline usage.

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by magnattic** [comment]
Same question, did you ever find the answer to this?
